src/ui_fsmenu/addons/manager.cc

```diff
@@ -1604,9 +1575,9 @@ void AddOnsCtrl::autofix_dependencies() {
 		if (!found) {
 			UI::WLMessageBox w(
 			   &get_topmost_forefather(), UI::WindowStyle::kFsMenu, _("Error"),
-			   (boost::format(_("The required add-on ‘%s’ could not be found on the server.")) %
+			   bformat(_("The required add-on ‘%s’ could not be found on the server.") ,
 			    addon_to_install)
-			      .str(),
+			      ,
```

Comment:
Now, why was this not corrected by `clang-format`?

(Sorry, couldn't make a suggestion, because of the deleted line in the diff.)

Reply:
This whole code block has been commented out with `#if 0` so it's treated like a comment